package.json

```diff
@@ -22,17 +22,17 @@
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.3.1",
     "@mui/material": "^5.3.1",
-    "next": "^12.0.9",
-    "react": "17.0.2",
+    "next": "^13.3.0",
```

Comment:
would be beneficial to stay with original versions of Next, React, etc.. would recommend omitting this file unless there's a feature that is not currently implemented in the previous versions

Reply:
Oh okay, I will try omitting the file. I think the versions got updated after running yarn or npm.